safe/gui/tools/wizard/step_kw57_extra_keywords.py

```diff
@@ -0,0 +1,202 @@
+# coding=utf-8
+"""InaSAFE Wizard Step Extra Keywords."""
+
+from PyQt4.QtGui import (
+    QLineEdit, QDateTimeEdit, QDoubleSpinBox, QComboBox, QCheckBox
+)
+from PyQt4.QtCore import Qt
+from collections import OrderedDict
+
+from datetime import datetime
+import pytz
+from safe import messaging as m
+from safe.definitions.hazard import hazard_volcanic_ash
+from safe.definitions.extra_keywords import (
+    extra_keyword_volcano_name,
+    extra_keyword_volcano_alert_level,
+    extra_keyword_eruption_height,
+    extra_keyword_volcano_eruption_event_time,
+    extra_keyword_time_zone,
+)
+from safe.gui.tools.wizard.wizard_step import WizardStep
+from safe.gui.tools.wizard.wizard_step import get_wizard_step_ui_class
+from safe.utilities.i18n import tr
+from safe.common.custom_logging import LOGGER
+
+__copyright__ = "Copyright 2017, The InaSAFE Project"
+__license__ = "GPL version 3"
+__email__ = "info@inasafe.org"
+__revision__ = '$Format:%H$'
+
+FORM_CLASS = get_wizard_step_ui_class(__file__)
+
+
+class StepKwExtraKeywords(WizardStep, FORM_CLASS):
+
+    """InaSAFE Wizard Step Source."""
+
+    def __init__(self, parent=None):
+        """Constructor for the tab.
+
+        :param parent: parent - widget to use as parent (Wizard Dialog).
+        :type parent: QWidget
+
+        """
+        WizardStep.__init__(self, parent)
+        self.widgets_dict = OrderedDict()
+
+    def is_ready_to_next_step(self):
+        """Check if the step is complete. If so, there is
```

Comment:
According to PEP, first line of docstring must finish with full stop on a single line

Reply:
Done.